src/codegen/mod.rs

```diff
@@ -49,6 +49,7 @@ use crate::{Entry, HashMap, HashSet};
 use std;
 use std::borrow::Cow;
 use std::cell::Cell;
+use std::collections::HashSet as CollectionHashSet;
```

Comment:
Please keep using `crate::HashSet`? Is there any reason not to?

Reply:
No reason, I was just having trouble using `HashSet::new()`. `Default::default()` works though.